strapi/strapi
# feat(content-releases): scheduling in multiple strapi instances

### What does it do?

This PR ensures that the publish process locks the release's row, preventing race conditions when multiple Strapi instances are running simultaneously. I recommend reading the documentation added in this PR to gain a better understanding of the problem and how this solution addresses it.

### How to test it?

You need to simulate multiple Strapi instances. You can achieve this using any technology of your choice, or simply by running the same Strapi application on multiple terminals with different ports.

1. Create a scheduled release and add at least one entry to it.
2. **Restart all Strapi applications.** This ensures that the cronjob for scheduling is created in all Strapi instances.
3. Only one instance should log the "Starting publishing" message in the console, and the webhook notification should be triggered only once.
